src/providers/DevicesProvider/docs/AudioOutputProvider.stories.mdx

```diff
@@ -48,13 +49,13 @@ const App = () => (
 
 const MyChild = () => {
   const { devices, selectedDevice } = useAudioOutputs();
-  const items = devices.map(device => (
+  const items = devices.map((device) => (
     <li key={device.deviceId}>{device.label}</li>
   ));
 
   return (
     <div>
-      <p>Current Selected DeviceId: {selectedDevice}</p>
+      <p>Current Selected Audio Output Device: {JSON.parese(selectedDevice)}</p>
```

Comment:
nit: Typo and same comments from AudioInputProvider.

Reply:
Also since audio output device is a string | null, we dont need the JSON.parse here.

Reply:
Updated.